File: BankKycSystemdotnet/BankKycSystem/DTOs/TransactionDTO.cs
using System;

namespace BankKycSystem.DTOs
{
    public class TransactionDTO
    {
        public long Id { get; set; }
        public decimal Amount { get; set; }
        public string Type { get; set; }
        public string? Description { get; set; }
        public DateTime Timestamp { get; set; }
        public string AccountNumber { get; set; }
    }
}

File: bank-kyc-system/src/main/java/com/bank/project/dto/TransactionDTO.java
package com.bank.project.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class TransactionDTO {
    private Long id;
    private BigDecimal amount;
    private String type;
    private String description;
    private LocalDateTime timestamp;
    private String accountNumber;
}
